tka/tka.go

```diff
@@ -714,3 +714,8 @@ func (a *Authority) Keys() []Key {
 	}
 	return out
 }
+
+// StateIDs returns the stateIDs for this tailnet key authority.
+func (a *Authority) StateIDs() (uint64, uint64) {
```

Comment:
// TODO(tom): document all this 

:)

Reply:
I've added a bit more verbaige, but most of the docs for stateID's are on the `State` struct. Theres more context on stateIDs specifically in https://tailscale.slack.com/archives/C03LXV1NTB8/p1668552550674699